app/src/main/res/layout/list_live_matches.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+              android:orientation="vertical">
+    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                  android:layout_width="match_parent"
+                  android:layout_height="match_parent"
+                  android:orientation="vertical">
+        <TextView android:id="@+id/title"
+                  android:layout_width="wrap_content"
+                  android:layout_height="wrap_content"
+                  android:textStyle="bold"/>
+    </LinearLayout>
+    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                  android:layout_width="match_parent"
+                  android:layout_height="match_parent"
+                  android:orientation="horizontal"
+                  android:weightSum="2">
+        <TextView
```

Comment:
Use databinding. Take a look at it and add a textview for each player. It will help you avoid the aforementioned string manipulation in the activity.

Reply:
Databinding sounds like something we might want to use eventually. I don't want to introduce it in this PR as another new feature. Also it seems to have some drawbacks seen [here](https://medium.com/@Miqubel/4-reasons-im-not-using-android-data-binding-e62127c2650c) for example

Reply:
Yes, databinding does have these features but we are unlikely to run into them on this screen particularly. This app is not complex enough to worry about scalability or the spahgetti code issue.

I do like the idea of using butterknife (although I don't see you using it) but I have never used it in tandem with a listview, only with recycler.

I would suggest reworking this to have a layout such as:

<Textview
id="@+id/player1"/>
<TextView
id="@+id/player2/>

Reply:
I'm using the Kotlin Android Extensions (basically Butter Knife for Kotlin) for convenient data binding now. Also reworked the layout to have an individual TextView for each player